src/dipali/Bank.java

```diff
@@ -0,0 +1,62 @@
+package dipali;
+
+public class Bank {
+	
+	int debitCnt=0 , creditCnt=0 , printCnt=0;
+	static int debitCount=0 , creditCount=0 , printCount=0;
+	int totalAmount=20000;
+	public void debitAmount(int amt) {
+		totalAmount=totalAmount-amt;
```

Comment:
what is requested amt is more then total amount, you have not handle this scenario.